tsconfig.json

```diff
@@ -23,6 +23,7 @@
     "baseUrl": ".",
     "useUnknownInCatchVariables": false,
     "paths": {
+        "@tiptap/core": ["node_modules/@tiptap/core/src/index"],
```

Comment:
Is this needed? I wouldn't expect this sort of import rewriting to be needed for this library. The fact that we do it for the other packages is something specific to the monorepo

Reply:
it was needed, but apparently they fixed it with the new version

Reply:
Nope, they did not ... the types get not resolved correctly and therefor the `check-types` pipeline is failing